addons/auth_totp/wizard/auth_totp_wizard_views.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_totp_wizard">
+        <field name="name">auth_totp wizard</field>
+        <field name="model">auth_totp.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="o_auth_totp_enable_2FA row container">
+                        <div class="mb-3">
+                            <h3 class="font-weight-bold">Authenticator App Setup</h3>
+                            <ul>
+                                <li>
+                                    <div class="d-flex align-items-center flex-wrap">
+                                        Install an Authenticator App
+                                        <div>
+                                            <a href="https://play.google.com/store/search?q=authenticator&amp;c=apps" class="mx-2" target="blank">
+                                                <img alt="On Google Play" style="width: 120px;" src="/base_setup/static/src/img/google_play.png"/>
+                                            </a>
+                                            <!--TODO DBE : find the correct url and the correct class-->
+                                            <a href="https://www.apple.com/be-fr/search/authenticator?src=globalnav" class="mx-2" target="blank">
```

Comment:
I don't think `be-fr` is the default we want worldwide

Reply:
oupsie, but anyway, I'm still not sure this link is the correct one. I don't have an iphone so I cannot test it ! #genius!

Reply:
You probably need `sel=explore` to ensure it opens the "explore" category, otherwise it might decide to open the Accessories section instead if it finds something there (that's exactly the result I had opening the link, I was a bit surprised to see a printer).